File: Utkarsh mishra/Arrays and String/Questions/containsduplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> s=new HashSet<>();
        for(int i=0;i<nums.length;i++)
            s.add(nums[i]);
        if(nums.length==s.size())
            return false;
        else
            return true;
        
    }
}

File: Shreya/Arrays and Strings/Contains Duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map <int,int>mp;
        for(int i=0;i<nums.size();i++)
        {
            if (mp[nums[i]]==0)
            {
                mp[nums[i]]=1;
            }
            else
            {
                mp[nums[i]]+=1;
                return true;
            }
        }
        return false;
    }
};